http-clients/netty-nio-client/src/test/java/software/amazon/awssdk/http/nio/netty/internal/FutureCancelHandlerTest.java

```diff
@@ -78,7 +78,6 @@ public void methodSetup() {
         when(ctx.channel()).thenReturn(channel);
         when(channel.attr(EXECUTION_ID_KEY)).thenReturn(attrMap.attr(EXECUTION_ID_KEY));
         when(channel.attr(REQUEST_CONTEXT_KEY)).thenReturn(attrMap.attr(REQUEST_CONTEXT_KEY));
-        when(channel.id()).thenReturn(DefaultChannelId.newInstance());
```

Comment:
Why did you remove this line from the code?

Reply:
Because I got this error

```
[ERROR] Errors:
[ERROR]   FutureCancelHandlerTest » UnnecessaryStubbing
Unnecessary stubbings detected in test class: FutureCancelHandlerTest
Clean & maintainable test code requires zero unnecessary code.
Following stubbings are unnecessary (click to navigate to relevant line of code):
  1. -> at software.amazon.awssdk.http.nio.netty.internal.FutureCancelHandlerTest.methodSetup(FutureCancelHandlerTest.java:81)
Please remove unnecessary stubbings or use 'lenient' strictness. More info: javadoc for UnnecessaryStubbingException class.
[INFO]
[ERROR] Tests run: 401, Failures: 0, Errors: 1, Skipped: 1
```